.github/workflows/linux-build.yml

```
@@ -44,15 +40,71 @@ on:
+        run: |
+          git fetch
+      - name: Install Core Dependencies
+        run: |
+          source scripts/setup-ubuntu.sh && install_apt_deps
+      - name: Install Docs Dependencies
+        run: |
+          sudo apt install -y --no-install-recommends pandoc
+          sudo apt install -y pipx
+          pipx install --force uv cmake==3.31
+          make python-venv
+          ~/.local/bin/uv pip install -r scripts/docs-requirements.txt 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)